src/dodal/devices/watsonmarlow323_pump.py

```diff
@@ -0,0 +1,61 @@
+from enum import Enum
+
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.signal import epics_signal_rw
+
+
+class WatsonMarlow323PumpEnable(str, Enum):
+    DISABLED = "Disabled"
+    ENABLED = "Enabled"
+
+
+class WatsonMarlow323PumpDirection(str, Enum):
+    CLOCKWISE = "CW"
+    COUNTER_CLOCKWISE = "CCW"
+
+
+class WatsonMarlow323PumpState(str, Enum):
+    STOPPED = "STOP"
+    STARTED = "START"
+
+
+class WatsonMarlow323Pump(StandardReadable):
+    """Watson Marlow 323 Pump device"""
+
+    def __init__(self, prefix: str, name: str = "") -> None:
+        self.enabled = epics_signal_rw(
+            WatsonMarlow323PumpEnable,
+            read_pv=prefix + "DISABLE",
+            write_pv=prefix + "DISABLE",
```

Comment:
write_pv isn't required if they're both the same.

Reply:
Fixed in 6d5ae3fb866bdad8de27966fc11605b6c41856cd